WORKING_GROUPS.md

```diff
@@ -0,0 +1,247 @@
+# Node.js Working Groups
+
+Node.js Working Groups are autonomous projects created by the
+[Technical Steering Committee (TSC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+
+Working Groups can be formed at any time but must be ratified by the TSC.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the TSC.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group charter is completed the Working
+Group should be dissolved and the responsibility for governance absorbed
+back in to the TSC.
+
+## Current Working Groups
+
+> (none)
+
+Technical Working Groups exist in the [nodejs/node repository](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md). 
+
+#### Process:
+
+* [Starting a Working Group](#starting-a-wg)
+* [Bootstrap Governance](#bootstrap-governance)
+
+## Starting a WG
+
+A Working Group is established by first defining a charter  that can be
+ratified by the TC. A charter is a *statement of purpose*, a
+*list of responsibilities* and a *list of initial membership*.
+
+A working group needs 3 initial members. These should be individuals
+already undertaking the work described in the charter.
+
+The list of responsibilities should be specific. Once established, these
+responsibilities are no longer governed by the TC and therefore should
+not be broad or subjective. The only recourse the TC has over the working
+group is to revoke the entire charter and take on the work previously
+done by the working group themselves.
+
+If the responsibilities described in the charter are currently
+undertaken by another WG then the charter will additionally have to be
+ratified by that WG.
+
+You can submit the WG charter for ratification by sending
+a Pull Request to this document, which adds it to the
+list of current Working Groups. Once ratified the list of
+members should be maintained in the Working Group's
+README.
+
+## Bootstrap Governance
+
```

Comment:
There does not appear to be any mention of moderation policy. That should be discussed here also.